### PR TITLE
Scikit

### PR DESCRIPTION
**Pull request by mowill**
This is an alteration of the python side of the dmdtools package that uses a scikit-like interface.

### CODE REVIEW

File: python/dmdtools/batch.py [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+        if Gy:
+            return self(X, X), self(Y, X), self(Y, Y)
+        else:
+            return self(X, X), self(Y, X)
+
+
+def sort_modes_evals(dmd_class, k=None, sortby="LM", target=None):
+    """ Sort and return the DMD or KDMD modes and eigenvalues
+
+    Paramters
```
Comment (cwrowley) [inline python/dmdtools/batch.py]: 'Paramters' is misspelled -- gives an error when run through numpydoc

File: python/dmdtools/batch.py [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+
+        See Tu et al., 2014 for details.
+
+    total : bool, optional
+        If false (default), compute the ``standard'' DMD modes
+        If true, compute the total least squares DMD modes
+
+        See Hemati & Rowley, 2015 for details.
+
+    Properties
```
Comment (cwrowley) [inline python/dmdtools/batch.py]: 'Properties' is not an allowed section heading in numpydoc.  Should probably be 'Attributes'

**Comment by cwrowley** [comment]
Looks great.  @mowill do you want to make those small changes in the docstrings, and I'll merge this in?

**Comment by mowill** [comment]
Hi Clancy,

Hope things are going well.  I’ll make these changes (probably over the weekend when I have some time), and let you know when I think it is ready to be merged in.  Thanks!
- Matt

On Aug 11, 2015, at 12:28 PM, Clancy Rowley <notifications@github.com<mailto:notifications@github.com>> wrote:

Looks great. @mowillhttps://github.com/mowill do you want to make those small changes in the docstrings, and I'll merge this in?

**Comment by mowill** [comment]
I've made the changes to the docstrings.  In addition to what you pointed out, my version of numpydoc didn't like latex-like quotation marks (things like ``quote'').  It didn't cause an error, but it created a very large number of warnings.  I've replaced them with standard quotes. 

@cwrowley would you mind taking another look and merging if things look good?  Thanks!

**Pull request merged**